pkg/server/datastore/sqlstore/mysql.go

```diff
@@ -61,6 +64,10 @@ func (my mysqlDB) connect(cfg *configuration, isReadOnly bool) (db *gorm.DB, ver
 		return nil, "", false, err
 	}
 
+	if version == "5.7" {
```

Comment:
Since we are getting the version running a `SELECT VERSION()` query, isn't the version string the complete version number, like 5.7.44?
I'm thinking that this should probably check if the version _starts with_ rather than comparing with a complete string.

Reply:
Yeah, good catch. I changed the logic to check for the version prefix "5.7." so we can handle any 5.7 versions.